docs/source/configuration/experimental.md

```diff
@@ -0,0 +1,35 @@
+---
+myst:
+  html_meta:
+    "description": "Enable experimental features in Volto's configuration object"
+    "property=og:description": "Enable experimental features in Volto's configuration object"
+    "property=og:title": "Experimental features"
+    "keywords": "Volto, Plone, frontend, experimental"
```

Comment:
```suggestion
    "keywords": "Volto, Plone, frontend, experimental, features"
```

Reply:
@stevepiercy Sure, we can add this. Is it generally necessary to put words in keywords that are already in the title?

Reply:
They are converted to meta tags. Just standard SEO stuff.